apps/web/contexts/Analytics.tsx

```diff
@@ -41,10 +42,10 @@ export default function AnalyticsProvider({ children, context }: AnalyticsProvid
       const sanitizedEventName = eventName.toLocaleLowerCase();
       if (typeof window === 'undefined') return;
       logEvent(
-        sanitizedEventName, // TODO: Do we want context here?
+        sanitizedEventName,
         {
           action: action,
-          context: fullContext,
+          context: fullContext as AnalyticsContext,
```

Comment:
this isn't technically correct. `fullContext` can be a concatenation of multiple contexts, so it's not necessarily a member of this type